### PR TITLE
Wiki restructuring - interoperability

### PR DESCRIPTION
**Pull request by eriqua**
# Change

Added interoperability section

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: docs/wiki/Home.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ If you're unfamiliar with Infrastructure as Code, or wonder how you can use the
   - [Pipeline usage](./The%20CI%20environment%20-%20Pipeline%20usage)
 - [Interoperability](./Interoperability)
   - [Bicep to ARM/JSON conversion](./Interoperability%20-%20Bicep%20to%20ARM%20conversion)
-  - [Register ADO pipelines](./Interoperability%20-%20Register%20ADO%20pipelines)
+  - [Transition to Azure DevOps Pipelines](./Interoperability%20-%20Transition%20to%20Azure%20DevOps%20Pipelines)
```
Comment (MrMCake) [inline docs/wiki/Home.md:62]: Why 'Transition to'? The ADO pipelines are already there, so a user would just need to register them.

File: docs/wiki/Interoperability - Transition to Azure DevOps Pipelines.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+Not all customers may be using GitHub repos and/or GitHub Actions. CARML is able to be hosted in a GitHub repo and deployed with Azure DevOps Pipelines or it can completely be ran out of Azure DevOps.
```
Comment (MrMCake) [inline docs/wiki/Interoperability - Transition to Azure DevOps Pipelines.md:1]:
```suggestion
Not all customers may be using GitHub repos and/or GitHub Actions. CARML can be hosted in a GitHub repo and deployed using Azure DevOps Pipelines or run entirely through Azure DevOps.
```

File: docs/wiki/Interoperability.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+This section explores how CARML is able to operate between ARM JSON Templates & Bicep and Azure DevOps Pipelines & GitHub Action to meet different customer scenarios.
```
Comment (MrMCake) [inline docs/wiki/Interoperability.md:1]: This section explores how CARML can work with both Bicep & ARM JSON templates as well as GitHub Actions & Azure DevOps Pipelines to meet different customer scenarios.
```suggestion
This section explores how CARML is able to operate between ARM JSON Templates & Bicep and Azure DevOps Pipelines & GitHub Action to meet different customer scenarios.
```

File: docs/wiki/Home.md (hunk 1 above)
Comment (eriqua) [reply docs/wiki/Home.md:62]: Comes from the lab. I'll revert it back to Register

Comment (eriqua) [reply docs/wiki/Home.md]: dan

**Review by MrMCake — APPROVED** [review]

**Pull request merged**